pkg/landscaper/execution/reconcile.go

```diff
@@ -152,6 +155,7 @@ func (o *Operation) deployOrTrigger(ctx context.Context, item executionItem) err
 		item.DeployItem = &lsv1alpha1.DeployItem{}
 		item.DeployItem.GenerateName = fmt.Sprintf("%s-%s-", o.exec.Name, item.Info.Name)
 		item.DeployItem.Namespace = o.exec.Namespace
+		item.DeployItem.Generation = o.exec.Generation
```

Comment:
the generation of a resource is updated by the k8s api server and cannot be updated by us.
Also the deploy items generation does not need to have the same generation as its execution.
```suggestion
```